src/mito2/src/cache/file_cache.rs

```diff
@@ -100,34 +100,45 @@ impl FileCache {
         self.memory_index.insert(key, value).await;
     }
 
+    pub(crate) async fn get_reader(&self, file_path: &str) -> ObjectStoreResult<Option<Reader>> {
```

Comment:
This method doesn't need to be `pub(crate)`. This result is not commonly used so we can use a qualified version.
```suggestion
    async fn get_reader(&self, file_path: &str) -> object_store::Result<Option<Reader>> {
```